packages/app/obojobo-express/routes/api/media.js

```diff
@@ -1,26 +1,24 @@
 const express = require('express')
 const router = express.Router()
 const multer = require('multer')
-
 const mediaConfig = oboRequire('config').media
 const MediaModel = oboRequire('models/media')
-
-const diskStorage = multer.diskStorage({
-	destination: mediaConfig.tempUploadDestination
-})
-
-// single() takes the field name of the file field
-//  in the multipart form being submitted
-const multerUpload = multer({
-	storage: diskStorage,
-	limits: {
-		fileSize: mediaConfig.maxUploadSize
-	}
-}).single('userImage')
-
 const { requireCurrentUser } = oboRequire('express_validators')
 
 const upload = (req, res) => {
+	const diskStorage = multer.diskStorage({
```

Comment:
multer and diskStorage initialization moved inside the upload function so it can be garbage collected. Small optimization trading speed for ram